omni_authify/frameworks/drf.py

```diff
@@ -39,10 +39,14 @@ def get_provider(self, provider_name, provider_settings):
                     fields=provider_settings.get('fields'),
                 )
 
-            # case 'google':
-            #     return Google(
-            #
-            #     )
+            case 'google':
```

Comment:
`get_provider` method has been moved to `omni_authify/core/utils.py` location to follow the **python's DRY rules**. Please, update the `get_provider` with your key for the provider google and then call the method to `omni_authify/frameworks/django.py` to use it.

Example: Take a look at my code: [django.py](https://github.com/Omni-Libraries/omni-authify/blob/main/omni_authify/frameworks/django.py)

![Uploading Screenshot 2024-12-14 at 16.05.56.png…]()